net/rsync/Makefile

```diff
@@ -69,6 +69,15 @@ define Package/rsyncd
   URL:=https://rsync.samba.org/
 endef
 
+define Package/rrsync
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=File Transfer
+  TITLE:=Restricted rsync script
+  DEPENDS:=+rsync +perlbase-file +perl
```

Comment:
As to me, it should also depend on openssh-server since dropbear doesn't support ~/.authorized_keys, but I don't have a strong opinion on this.

Reply:
That makes sense, though it could depend on either openssh-server or openssh-server-pam. So should it be this?

```suggestion
  DEPENDS:=+rsync +perlbase-file +perl @(PACKAGE_openssh-server||PACKAGE_openssh-server-pam)
```

Reply:
Depending on either one makes sense, but I'm not an expert in dependencies like this, and my build host is down atm so I can't check it myself.
If this expression does the job, I'm good with it.

Reply:
It works - if I don't select either openssh-server or openssh-server-pam then rrsync does not appear in menuconfig. If I select one or other, or both, then it appears.

Reply:
I was also a bit unsure about this line. However, similar things are done i pppossh and prometheus-node-exporter-lua.